### DISCUSSION TITLE
Fixes for the docs

### DISCUSSION THREAD

**Discussion by tillschander** [tobimori/kirby-dreamform#73] [General] [answered]
Hi,

I have found some errors in the docs. Usually I would send some PRs with fixes. But the docs don't seem to be open source.

Is there an other place where I (and others) can suggest improvements to the docs?

Till

**Comment by tobimori** [comment, marked as answer]
Send me an email: support@andkindness.com
Thanks for your help

**Reply by tillschander** [reply]
Will do
